lib/Connection.js

```diff
@@ -218,7 +218,7 @@ class Connection {
       origin: url.origin,
       // https://github.com/elastic/elasticsearch-js/issues/843
```

Comment:
I dislike comments like these. I say let's add a unit test and we don't need to explain.

Reply:
agreed. Also, original fork for this PR was deleted, so I'm going to close this PR and open a new one, and will add a unit test.